Pixel Game/Assets/Scripts/ActiveQuests.cs

```diff
@@ -46,4 +48,19 @@ public void TryIncrementKilledEnemies(string name)
         foreach (var quest in quests)
             quest.TryIncrementKilledEnemies(name);
     }
+
+    //Lets you use ActiveQuests objects in foreach loops
```

Comment:
So the getEnumerator gets the AMOUNT of current active quests? :)

Reply:
Well it specifies the sequence of objects to iterate over in foreach loops.  In this case we use the predefined enumerator in the List class which "yield returns" all objects in the list, one per iteration.
So using `quests.GetEnumerator` means that writing
```c#
foreach (var quest in activeQuests)
```
is the same as
```c#
foreach (var quest in activeQuests.quests)
```

Reply:
Aren't they the same? :D